src/XmlStringStreamer/Stream/File.php

```diff
@@ -16,13 +16,8 @@ public function __construct($mixed, $chunkSize = 16384, $chunkCallback = null)
             // Account for common stream/URL wrappers before checking if a file exists
             $realPath = $mixed;
             if (preg_match('/^([\w.]+):\/\//', $realPath, $matched)) {
-                if (preg_match('/(http|ftp|php|data|ssh2)/', $matched[1])) {
-                    // Disable file_exists() check
-                    $realPath = null;
-                } else {
-                    // Remove wrapper for file_exists() check.
-                    $realPath = substr($realPath, strlen($matched[0]));
```

Comment:
Hm, I feel something will be lost here:

https://github.com/prewk/xml-string-streamer/commit/8bfccd2b3e899858d3f4fa33123a8c3b0f721be0

Maybe @friartuck6000 could shed some light on this case:

* `$mixed` has a protocol (starts with `aword://`)
* ..but doesn't match `http|ftp|php|data|ssh2`

Why are we removing the protocol in that case? Is it for a real use case?

Reply:
TBH I think the logic is a bit odd.  I don't see much of a point to remove the wrapper, because it's a problem by the caller (could've removed the protocol if he wants it to load from the server locally before passing into the library)

Removing it will make more logical sense.

Reply:
I agree, let's make it more permissive.